pkg/metrics/metrics.go

```diff
@@ -10,10 +10,17 @@ var (
 		Name:      "reconfigured_total",
 		Help:      "Total times the image registry's storage was reconfigured.",
 	})
+	imagePrunerInfo = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "image_registry_operator",
+		Subsystem: "image_pruner",
+		Name:      "install_info",
```

Comment:
We tend to simply do the entire thing in name these days. I think the other fields might even be deprecated going forward.
`Name: image_registry_operator_image_pruner_install_info`.